### DISCUSSION TITLE
Squad Post/Share API

### DISCUSSION THREAD

**Discussion by ncosentino** [dailydotdev/daily#1241] [Feature Requests]
As a squad organizer, it would be very be beneficial to have the ability to programmatically share to the application. I'm currently posting multiple times per day on the platform and have grown my squad to over several thousand members. The content is being engaged with, but this would help dramatically with my ability to keep up.

**Comment by idoshamun** [comment]
Hi @ncosentino,

Thanks for the suggestion.
Public API is a requested feature that unfortunately for us is currently out of scope.
On the surface it's seems like a simple task but given the speed we're moving at currently maintaining a stable API will be a challenge.
Pretty sure we will release an API at some point 🙏
